### DISCUSSION TITLE
Remote Temperature Timeout?

### DISCUSSION THREAD

**Discussion by CopperSubstrate** [echavet/MitsubishiCN105ESPHome#202] [Q&A] [answered]
How does the remote temperature timeout work? I have Centealite temp sensors that usually report temperate every 5 minutes. Yet, during periods in which updates are received, the internations seem to sometimes jump back to the internal temp readings. Since the internal temp readings in winter are so inaccurately high, I can see the unit temp jump in the logs when this happens. I'm not sure why this is happening.

**Comment by tlut** [comment, marked as answer]
You could try adding
`- heartbeat: 120s`
under sensor -> platform -> filters. 

Had the same problem, and only been running this setup for a couple of hours, but no issues since.

I'm not sure if this then defeats the whole remote_temperature_timeout.
